.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
 - rm -rf node_modules
 - npm install
 - npm install web3@1.0.0-beta.37 ethereumjs-util ethereumjs-abi bignumber.js --no-save
+- npm install --global typescript
```

Comment:
TypeScript is already installed in the project with `npm install`, I think [adding tsc to build](https://github.com/ccxt/ccxt/pull/6510/commits/2377a5a85f174a57615c722861af5c10cc997ce9) should take care of the rest.